src/ui/units/collections/components/CollectionActions/CollectionActions.tsx

```diff
@@ -49,17 +49,12 @@ export const CollectionActions = React.memo<Props>(
         onDeleteClick,
     }) => {
         const collection = useSelector(selectCollection);
-        const rootCollectionPermissions = useSelector(selectRootCollectionPermissions);
 
         const {CustomActionPanelCollectionActions} = registry.collections.components.getAll();
 
-        const showCreateCollection = collection
-            ? collection.permissions?.createCollection
-            : rootCollectionPermissions?.createCollectionInRoot;
+        const showCreateCollection = collection ? collection.permissions?.createCollection : true;
 
-        const showCreateWorkbook = collection
-            ? collection.permissions?.createWorkbook
-            : rootCollectionPermissions?.createWorkbookInRoot;
+        const showCreateWorkbook = collection ? collection.permissions?.createWorkbook : true;
```

Comment:
So now we can showAddDemoWorkbook if we can't create workbook

Reply:
Yes, user will see "Demo workbook" button